### DISCUSSION TITLE
Operation of the blurscreen option

### DISCUSSION THREAD

**Discussion by peavine** [swiftDialog/swiftDialog#455] [Q&A] [answered]
Either I don't understand the operation of the `--blurscreen` option, or I'm doing something wrong. When I run the following command in a terminal window, the dialog appears but the screen behind it is a solid gray. I'm running swiftDialog 2.5.3 beta 2 and Sequoia 15.1. Thanks!

```
dialog --title "Test Title" --message "Test Message" --blurscreen --moveable
```

**Comment by bartreardon** [comment]
you're not doing anything wrong. depends what you have in the background. if there's a lot of nothing behind then it might appear like a solid colour.  Also, there can be issues if you switch spaces but that's more of an OS thing and how layering works.

**Comment by BigMacAdmin** [comment, marked as answer]
The solid grey background is the result of this setting in macOS: Settings > Accessibility > Display > Reduce Transparency

<img width="487" alt="image" src="https://github.com/user-attachments/assets/360d6651-c7c3-4766-9862-8a0d16b27dbe">

**Comment by bartreardon** [comment]
aah, I forget about that setting.

yeah, if reduce transparency is on then the transparent layer won't be.

**Comment by peavine** [comment]
Thanks. I turned off the Reduce transparency setting, and the `--blurscreen` option worked as expected.
